docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -122,6 +122,8 @@ Preparing For a Release
       mkvirtualenv flocker-release
       pip install --editable .[dev]
       admin/create-release-branch --flocker-version="${VERSION}"
+      admin/update-license
+      git commit -am "Updated copyright in LICENSE file"
```

Comment:
No, that would actually create an empty commit, and suggets that the copyright was updated when it wasn't. The behaviour of `git` is fine as is. It is just that a future automated script will need to deal with this returning non-zero.

Maybe have `update-licence` return a specific exit code meaning, "no error, but also no update". This can be ignored by the release engineer now, but would help develop a solid automated script later.